FullJournalImportExportPlugin.inc.php

```diff
@@ -308,7 +308,7 @@ function _removeTemporaryFiles($tempfiles) {
 	function tarFiles($targetPath, $targetFile, $sourceFiles) {
 		assert($this->_checkedForTar);
 
-		$tarCommand = Config::getVar('cli', 'tar') . ' -czf ' . escapeshellarg($targetFile);
+		$tarCommand = Config::getVar('cli', 'tar') . ' -cJf ' . escapeshellarg($targetFile);
```

Comment:
There's no guarantee that a particular tar supports xz compression, whereas gzip compression is (nearly) universal.  See, for example, Solaris tar.  Support should either be detected, or should at the choice of the user.